tests/rustdoc/target-feature.rs

```diff
@@ -1,3 +1,5 @@
+#![feature(doc_cfg)]
+
```

Comment:
Does this mean `target_feature` will no longer result in any note on stable rustdoc? Or will we still display the attribute itself?

Reply:
That's probably acceptable for now given that our current note for `target_feature` is quite bad actually.  We'll just want to make sure that gets fixed before next stable.

Reply:
Sounds good to me.